recipes/python-lmdb/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "python-lmdb" %}
+{% set pypi_name = "lmdb" %}
+{% set github_name = "py-lmdb" %}
+{% set version = "0.92" %}
+{% set sha256 = "e3eaf5f4a4d0fbae9c99bc803c21f91636c2c68dfd42ab1d58284ba4883695fb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Note: we're using the GitHub tarballs as the PyPI source distribution
+  # doesn't include the test suite.
```

Comment:
No need for an explanation. We decided a long time ago that both are valid sources.